### PR TITLE
Fs update trie bench and reference trie

### PR DESCRIPTION
**Pull request by montekki**
1) Benches in `trie-db` look broken (running `cargo check --benches --all` seems to fail bc. a `small_rng` feature is not enabled for `rand`)
2) Update all depencencies on `trie-db` and `memory-db` to the latest ones.

### CODE REVIEW

File: trie-db/Cargo.toml [hunk 1]
```diff
@@ -8,15 +8,15 @@ license = "Apache-2.0"
 
 [dependencies]
 log = "0.4"
-rand = { version = "0.7", default-features = false }
+rand = { version = "0.7", default-features = false, features = [ "small_rng" ] }
```
Comment (ordian) [inline trie-db/Cargo.toml]: now that I think about it, should it be only a dev dependency?

Comment (montekki) [reply trie-db/Cargo.toml]: Doesn't [look](https://github.com/paritytech/trie/blob/master/trie-db/src/lib.rs#L23) so.

Comment (ordian) [reply trie-db/Cargo.toml]: But is it actually **used**?

Comment (montekki) [reply trie-db/Cargo.toml]: right, I think it's best to remove this change from this PR at all and move it to dev dependencies in #67

**Comment by montekki** [comment]
@ordian i think that the part of this PR that updates the deps should not have been dropped though

**Comment by ordian** [comment]
> @ordian i think that the part of this PR that updates the deps should not have been dropped though

I don't think it's necessary though. The bump from `0.19.0` to `0.19.2` is semver compatible, which means the reverse dependencies can be updated automatically by updating `Cargo.lock` via `cargo update`. Imagine `rand` bumping it's version from `0.7.1` to `0.7.2`, does it warrant updating versions of all crates that depend on `0.7.1`?

**Pull request closed without merging**